nodejs-webpack/nodejs-webpack.spec

```diff
@@ -0,0 +1,533 @@
+Provides: bundled-npm(string-width) = 1.0.1
+Provides: bundled-npm(wide-align) = 1.1.0
+Provides: bundled-npm(isobject) = 2.1.0
+Provides: bundled-npm(randomatic) = 1.1.5
+Provides: bundled-npm(is-number) = 2.1.0
+Provides: bundled-npm(strip-ansi) = 3.0.1
+Provides: bundled-npm(delayed-stream) = 1.0.0
+Provides: bundled-npm(readable-stream) = 2.0.6
+Provides: bundled-npm(assert-plus) = 0.2.0
+Provides: bundled-npm(hoek) = 2.16.3
+Provides: bundled-npm(boom) = 2.10.1
+Provides: bundled-npm(jsprim) = 1.3.0
+Provides: bundled-npm(sntp) = 1.0.9
+Provides: bundled-npm(sshpk) = 1.9.2
+Provides: bundled-npm(mime-db) = 1.23.0
+Provides: bundled-npm(commander) = 2.9.0
+Provides: bundled-npm(pinkie-promise) = 2.0.1
+Provides: bundled-npm(ms) = 0.7.1
+Provides: bundled-npm(is-my-json-valid) = 2.13.1
+Provides: bundled-npm(cryptiles) = 2.0.5
+Provides: bundled-npm(wrappy) = 1.0.2
+Provides: bundled-npm(fs.realpath) = 1.0.0
+Provides: bundled-npm(chalk) = 1.1.3
+Provides: bundled-npm(code-point-at) = 1.0.0
+Provides: bundled-npm(is-fullwidth-code-point) = 1.0.0
+Provides: bundled-npm(inflight) = 1.0.5
+Provides: bundled-npm(ansi-regex) = 2.0.0
+Provides: bundled-npm(json-schema) = 0.2.2
+Provides: bundled-npm(asn1) = 0.2.3
+Provides: bundled-npm(extsprintf) = 1.0.2
+Provides: bundled-npm(assert-plus) = 1.0.0
+Provides: bundled-npm(getpass) = 0.1.6
+Provides: bundled-npm(dashdash) = 1.14.0
+Provides: bundled-npm(jsbn) = 0.1.0
+Provides: bundled-npm(jodid25519) = 1.0.2
+Provides: bundled-npm(verror) = 1.3.6
+Provides: bundled-npm(ecc-jsbn) = 0.1.1
+Provides: bundled-npm(pinkie) = 2.0.4
+Provides: bundled-npm(tweetnacl) = 0.13.3
+Provides: bundled-npm(graceful-readlink) = 1.0.1
+Provides: bundled-npm(generate-function) = 2.0.0
+Provides: bundled-npm(generate-object-property) = 1.2.0
+Provides: bundled-npm(jsonpointer) = 2.0.0
+Provides: bundled-npm(escape-string-regexp) = 1.0.5
+Provides: bundled-npm(ansi-styles) = 2.2.1
+Provides: bundled-npm(xtend) = 4.0.1
+Provides: bundled-npm(has-ansi) = 2.0.0
+Provides: bundled-npm(number-is-nan) = 1.0.0
+Provides: bundled-npm(supports-color) = 2.0.0
+Provides: bundled-npm(is-property) = 1.0.2
+Provides: bundled-npm(qs) = 6.2.1
+AutoReq: no
+AutoProv: no
+
+
+%description
+Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jade, coffee, css, less, ... and your custom stuff.
+
+%package doc
+Summary: Documentation for nodejs-%{npm_name}
+Group: Documentation
+Requires: nodejs-%{npm_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+This package contains documentation for nodejs-%{npm_name}
+
+%prep
+mkdir npm_cache
+for tgz in %{sources}; do
+  echo $tgz | grep -q registry.npmjs.org || npm cache add --cache ./npm_cache $tgz
+done
+
+%setup -T -q -a 234 -D -n npm_cache
+
+%build
+npm install webpack@1.13.1 --cache-min Infinity --cache .  --verbose
```

Comment:
Prefer `%{npm_name}@%{version}` if possible, it'll reduce the changes required to update or between files.